evap/student/templates/student_index_evaluate_or_drop.html

```diff
@@ -0,0 +1,24 @@
+<a class="btn btn-sm btn-primary btn-row-hover" href="{% url 'student:vote' evaluation.id %}">
+    {% translate 'Evaluate' %}
+</a>
+{% if evaluation.allow_drop_out %}
+    <confirmation-modal>
+        <span slot="title">{% translate 'Did you drop this course?' %}</span>
+        <span slot="question">
+            {% blocktranslate %}
+                If you mark this course as dropped, you can still give feedback like normal, but your answers will only
+                be visible to contributors.
+            {% endblocktranslate %}
+            <br>
+            {% blocktranslate %}
+                You will not be able to vote for this course afterwards.
+            {% endblocktranslate %}
```

Comment:
I think this would be fine

```suggestion
            {% blocktranslate %}
                If you mark this course as dropped, you will still be able to provide feedback as usual. However, you can only submit the questionnaire once.
            {% endblocktranslate %}
```